edi_oca/tests/fake_models.py

```diff
@@ -7,10 +7,11 @@
 
 class EdiExchangeConsumerTest(models.Model):
     _name = "edi.exchange.consumer.test"
-    _inherit = ["edi.exchange.consumer.mixin"]
+    _inherit = ["edi.exchange.consumer.mixin", "edi.configuration.mixin"]
     _description = "Model used only for test"
 
     name = fields.Char()
+    partner_id = fields.Many2one("res.partner")
```

Comment:
why a partner_id? If there's no partner on the model we should not run anything or even break the init of the model if you are assigning configurations to a model w/o partners